README.md

```diff
@@ -47,19 +47,18 @@
   </a>
 </p>
 
-In essence Vocabulary is a collection of UI components, which are parts of 
+In essence Vocabulary is a collection of UI components, which are parts of
 web sites and web apps that have significance as being individual units that
-can be mixed, matched, styled and put together to quickly and effectively 
+can be mixed, matched, styled and put together to quickly and effectively
 develop consistent applications.
 
-
 ### Availability
 
 Vocabulary is available as
+
```

Comment:
Why the newline here? In my opinion keeping lists with the preceding paragraph improves readability.